sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/pipeline/_io/base.py

```diff
@@ -69,11 +69,21 @@ def __init__(self, meta: Union[Input, Output], data, **kwargs):
         :type data: Union[None, int, bool, float, str
                           azure.ai.ml.Input,
                           azure.ai.ml.Output]
+        :param original_data: default value of input/output, None means un-configured data.
+        :type original_data: Union[None, int, bool, float, str
+                          azure.ai.ml.Input,
+                          azure.ai.ml.Output]
         """
         self._meta = meta
+        self._original_data = data
         self._data = self._build_data(data)
+        self._default_data = default_data
```

Comment:
this is just to save it but no usage for now, right?

Reply:
Yes